zkevm-circuits/src/evm_circuit/execution/jumpi.rs

```diff
@@ -44,21 +44,10 @@ impl<F: Field> ExecutionGadget<F> for JumpiGadget<F> {
         let is_condition_zero = IsZeroGadget::construct(cb, phase2_condition.expr());
         let should_jump = 1.expr() - is_condition_zero.expr();
 
-        // Lookup opcode at destination when should_jump
-        cb.condition(should_jump.clone(), |cb| {
-            cb.require_equal(
-                "JUMPI destination must be within range if condition is non-zero",
-                dest.not_overflow(),
-                1.expr(),
-            );
-
-            cb.opcode_lookup_at(dest.valid_value(), OpcodeId::JUMPDEST.expr(), 1.expr());
-        });
-
         // Transit program_counter to destination when should_jump, otherwise by
         // delta 1.
         let next_program_counter = select::expr(
-            should_jump,
+            should_jump.clone(),
```

Comment:
Nit-pick: for consistency
```suggestion
            should_jump.expr(),
```